storage/innobase/xtrabackup/test/t/check_for_passwords.sh

```diff
@@ -0,0 +1,88 @@
+###############################################################################
+# check if for any password store in data dir thread info or in history table
+###############################################################################
+
+###############################################################################
+
+. inc/common.sh
+password_string=secretpwd
+
+MYSQLD_EXTRA_MY_CNF_OPTS="
+secure-file-priv=$TEST_VAR_ROOT
+"
+vlog "Preparing server"
+start_server
+mysql -e "CREATE USER pxb@localhost IDENTIFIED BY '$password_string'"
+mysql -e "GRANT ALL ON *.* TO pxb@localhost "
+mysql -e "CREATE TABLE t1(i int) engine=MYISAM" test
```

Comment:
explain why we need myisam engine table. What if someone tries to change to innodb.
Also we might need a relook when myisam gets removed